api/handlers/datasources.py

```diff
@@ -43,17 +43,18 @@ def add_imageURL_sasToken(datasource):
         and "sasToken" in datasource
         and datasource["sasToken"] is not None
         and datasource["sasToken"] != ""
-        and "core.windows.net" in datasource["imageURL"]
     ):
         # linter fix for error: "get_secret_value" is not a known member of "None" (reportOptionalMemberAccess)
         x = decrypt(datasource["sasToken"])
         y = ""
         if x is not None:
             y = x.get_secret_value()
-        imageURL_sasToken = SecretStr(datasource["imageURL"] + "?" + y)
+        
+        bloburl = f'https://{datasource["account"]}.blob.core.windows.net/{datasource["container"]}/image.jpg'
```

Comment:
I think the image url need to have a reference to the recording name itself no ?